### ISSUE TITLE
Build is broken on nightly

### ISSUE THREAD

**Issue by qti3e** [dfinity/cdk-rs#136]
Since the new Rust nightly release, the v0.4.0 of the num_bigint package is broken which candid is depended on, and since this crate is also depended on the Candid crate, this also fails to build on nightly.

I have already open a PR on the Candid repository to migrate to v0.4.2 of the num_bigint to fix the build errors, until then ic_cdk remains broken too.

Related links:
https://github.com/rust-lang/rust/issues/88581
https://github.com/rust-num/num-bigint/pull/219

**Comment by chenyan-dfinity** [comment]
I don't think it blocks the build. This is a minor version bump. You can run `cargo update && cargo build` or just delete `Cargo.lock`, it will work fine.

**Comment by qti3e** [comment]
Example build failure on docs.rs, caused due to this version bump:
https://docs.rs/crate/ic-kit/0.4.0/builds/432653

I managed to fix the build on local machine by putting this on my Cargo.toml:
```toml
[patch.crates-io]
num-bigint = { git = "https://github.com/rust-num/num-bigint" }
```

**Comment by adamspofford-dfinity** [comment]
No longer an issue as of latest nightly.
